core/modules/node/tests/node_layout_preview_revision_test/node_layout_preview_revision_test.module

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @file
+ * Node layout preview/revision test module.
+ */
+
+/**
+ * Implements hook_config_info().
+ */
+function node_layout_preview_revision_test_config_info() {
```

Comment:
I'm wondering if this hook implementation is really needed in the test?

Reply:
It's useful when you are manually testing out the actions with the test module, because the layouts get removed when the module is uninstalled. (By "manually testing," I mean "doing all the steps in the test manually", not running the tests from the UI.)